src/App.tsx

```diff
@@ -5,21 +5,22 @@ import { Routes, Route, Link } from 'react-router-dom'
 import { CollectionProvider } from 'hooks'
```

Comment:
you need to import and define GlobalStyles in your main index file:
https://github.com/ben-rogerson/twin.examples/tree/master/react-styled-components#add-the-global-styles